app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -31,7 +31,8 @@ def import_vm(source_vm_id, target_params)
   end
 
   def validate_import_vm
-    api_version >= '4.0'
+    version = highest_supported_api_version
```

Comment:
maybe we can use: (highest_supported_api_version is almost entirely memorized, if the caching fails it will return nil and not run twice anyway)
`highest_supported_api_version >= '4' if highest_supported_api_version`